### PR TITLE
Added GQL instructions for deleting org members

### PR DESCRIPTION
**Pull request by karensawrey**

### CODE REVIEW

File: pages/pipelines/permissions.md.erb [hunk 1]
```diff
@@ -114,3 +114,36 @@ From the Member Permissions page, organization admins can toggle whether or not
 - Disconnect agents
 
 If your organization has teams enabled, the pipeline creation permissions are managed at a team level. Pipeline creation permission controls can be found on the Teams Settings page. Without teams enabled, the pipeline creation permission control can be found on the Member Permissions page. 
+
+Organization admin can delete orgranization members. To do it, the admin needs to:
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:118]: Typo here (picked up by the linter).

Comment (plaindocs) [reply pages/pipelines/permissions.md.erb:118]: Also, we probably need to introduce the GraphQL element somehow. Maybe "To delete organization members using GraphQL, an organization admin needs to:" ?

File: pages/pipelines/permissions.md.erb [hunk 2]
```diff
@@ -114,3 +114,36 @@ From the Member Permissions page, organization admins can toggle whether or not
...
+  1. Find the ID for the user to be deleted ("Jane Doe"):  
+```bash
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:120]: Probs needs a blank line, and the code isn't bash. `graphql` should work nicely.

File: pages/pipelines/permissions.md.erb [hunk 3]
```diff
@@ -114,3 +114,36 @@ From the Member Permissions page, organization admins can toggle whether or not
...
+          }
+          id
+        }
+      }
+    }
+  }
+}
+```
+
+  2. Use the ID ("abc123") in a mutation:
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:138]: And this is differently indented and not fenced.

**Review by plaindocs — CHANGES_REQUESTED** [review]
If you expand the breaking build in the linter, you can often see an indication of what isn't right:

image

**Comment by karensawrey** [comment]
Oh my, such a small commit, so many issues %) Implemented all the fixes in one go. @plaindocs - please take a look

File: pages/pipelines/permissions.md.erb [hunk 4]
```diff
@@ -114,3 +114,41 @@ From the Member Permissions page, organization admins can toggle whether or not
 - Disconnect agents
 
 If your organization has teams enabled, the pipeline creation permissions are managed at a team level. Pipeline creation permission controls can be found on the Teams Settings page. Without teams enabled, the pipeline creation permission control can be found on the Member Permissions page. 
+
+Organization admin can delete organization members. To delete organization members using GraphQL, the admin needs to:
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:118]:
```suggestion
Organization admin can delete organization members. To delete organization members using GraphQL, the admin needs to:

```

File: pages/pipelines/permissions.md.erb [hunk 5]
```diff
@@ -114,3 +114,41 @@ From the Member Permissions page, organization admins can toggle whether or not
...
+  1. Find the ID for the user to be deleted ("Jane Doe"):
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:120]:
```suggestion
1. Find the ID for the user to be deleted ("Jane Doe"):  
```

File: pages/pipelines/permissions.md.erb [hunk 6]
```diff
@@ -114,3 +114,41 @@ From the Member Permissions page, organization admins can toggle whether or not
...
+          }
+          id
+        }
+      }
+    }
+  }
+}
+```
+
+  2. Use the ID ("abc123") in a mutation:
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:140]:
```suggestion
2. Use the ID ("abc123") in a mutation: 
```

Comment (plaindocs) [reply pages/pipelines/permissions.md.erb:140]: Oh, I suspect this is going to cause issue with the numbering, we might need to indent the first code block to fix that.

File: pages/pipelines/permissions.md.erb [hunk 7]
```diff
@@ -114,3 +114,41 @@ From the Member Permissions page, organization admins can toggle whether or not
...
+          user {
+            name
+          }
+          id
+        }
+      }
+    }
+  }
+}
+```
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:138]:
```suggestion
    ```graphql
    query {
    organization(slug: "your-organization-slug") {
        members(search: "Jane Doe", first: 10) {
        edges {
            node {
            role
            user {
                name
            }
            id
            }
        }
        }
    }
    }
    ```
```

File: pages/pipelines/permissions.md.erb [hunk 8]
```diff
@@ -114,3 +114,41 @@ From the Member Permissions page, organization admins can toggle whether or not
...
+    organization{
+      name
+    }
+    deletedOrganizationMemberID
+    user{
+        name
+    }
+  }
+}
+```
```
Comment (plaindocs) [inline pages/pipelines/permissions.md.erb:154]:
```suggestion
1. Find the `id` for the user to be deleted (in this example `Jane Doe`):  

    ```graphql
    query {
    organization(slug: "your-organization-slug") {
        members(search: "Jane Doe", first: 10) {
        edges {
            node {
            role
            user {
                name
            }
            id
            }
        }
        }
    }
    }
    ```

2. Use the `id` from the previous query in a mutation: 

    ```graphql 
    mutation deleteOrgMember {
    organizationMemberDelete(input: { id: "abc123" }){
        organization{
        name
        }
        deletedOrganizationMemberID
        user{
            name
        }
    }
    }
    ```
```

**Review by plaindocs — APPROVED** [review]
Apologies for messing around with your PR so much.

**Comment by karensawrey** [comment]
> Apologies for messing around with your PR so much.

Whatever it takes! :)

**Pull request merged**